src/testHelpers.tsx

```diff
@@ -17,3 +18,32 @@ export function renderWrapped(element: ReactElement): RenderResult {
     rerender: (newElement: ReactElement): void => renderResult.rerender(wrap(newElement)),
   };
 }
+
+export function aTestType(): TestType {
+  return {
+    id: 'some-test-type-id',
+    name: 'PCR test',
+    resultsSchema: {
+      title: 'Test result',
```

Comment:
We seem to require a `title` on the top level of `resultsSchema` as well, is it fine to keep it the same as the property one in our case?

Reply:
It's fine for an object schema to have a title.  When you say it's 'required' what do you mean, what requires it?

Reply:
Our current `ObjectSchema` definition (type of `testType.resultsSchema`). Would you make it optional instead?

Reply:
Yeah, it's definitely optional in the context of broader JSON schema.  There are a number of schemas that don't normally have a type: allOf, oneOf, anyOf.

Reply:
I'll make it optional here as well then — we're not using it for the `oneOf` case. Thanks!

Reply:
✅ Made it optional.